test/built-ins/Array/prototype/filter/resizable-buffer.js

```diff
@@ -0,0 +1,105 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.filter
+description: >
+  Array.p.filter behaves correctly on receivers backed by resizable
+  buffers
+includes: [compareArray.js, resizableArrayBufferUtils.js]
+features: [resizable-arraybuffer]
+flags: [onlyStrict]
```

Comment:
Do we need `onlyStrict` or is it leftover from the staging file?

Reply:
Done.